README.md

```diff
@@ -420,6 +421,69 @@ Style: 0 / 0
 Total: 100 / 100
 ```
 
+#### Proxy Resubmissions
+
+The autograder allows course staff to resubmit on behalf of students using "proxy" submissions.
+The proxy resubmit command is very similar to [proxy submit](#proxy-submissions) and includes the `proxy-email` and `proxy-time` parameters.
```

Comment:
I would make common section `#### Proxy Submissions`.
There you can cover the basics, lingo (e.g. define `proxy-email` and `proxy-time`),
and semantics (like rejection policies).


Then, you can have smaller more targeted section for each specific action, e.g.:
`##### Submitting Code Files for a Student` and `##### Resubmitting a Student's Code by ID`.